test/entt/signal/dispatcher.cpp

```diff
@@ -28,6 +28,19 @@ struct receiver {
     int cnt{0};
 };
 
+struct generic_receiver {
```

Comment:
I don't think we need to add a new class to test the `size` method. A bunch of calls to `size<T>` and `size` here and there should get the job done. It's pretty straightforward after all.